nat-lab/tests/test_vpn.py

```diff
@@ -228,7 +458,7 @@ async def test_vpn_connection(
         ),
     ],
 )
-async def test_vpn_reconnect(
+async def test_vpn_reconnect_wireguard(
```

Comment:
Couldn't that be parameterized with nordlynx and wireguard servers?

Reply:
:100: agree with @stalowyjez

Reply:
This is a good suggestion, after some back and worth I have it working and it's much nicer than it was. Thanks for the suggestion :)